cmd/chisel/cmd_info.go

```diff
@@ -0,0 +1,139 @@
+package main
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/jessevdk/go-flags"
+	"gopkg.in/yaml.v3"
+
+	"github.com/canonical/chisel/internal/setup"
+)
+
+var shortInfoHelp = "Show information about package slices"
+var longInfoHelp = `
+The info command shows detailed information about package slices.
+
+It accepts a white-space separated list of strings. The list can be
+composed of package names, slice names, or a combination of both. The
+default output format is YAML. When multiple package or slice names
+are provided, the output is a list of YAML documents, separated by
+three dashes (“---”).
+
+Slice definitions are shown verbatim according to their definition in
+the Chisel releases. For example, "globs" are not expanded.
```

Comment:
Nit: are they globs or are they "globs"? Quotes here seem to imply they're not real globs:
```suggestion
the Chisel releases. For example, globs are not expanded.
```